src/api/reference.md

```diff
@@ -1,83 +1,83 @@
 ---
-title: API Reference
+title: API 参考文档
 template: generic
 id: api/reference
 ---
 
-# API Reference
+# API 参考文档 ~~ API Reference
 
-This page details how to render Pug using the JavaScript API.
+此页面将详细说明如何 JavaScript API 来对 Pug 代码进行渲染。
 
-::: float info Tip
-Pug is available in your Web browser's console! To test drive Pug's API, as documented on this page, try entering:
+::: float info 提示
+现在 Pug 可以在您的浏览器控制台上使用！想测试各种 Pug 的 API，您可以尝试在控制台输入：
 
 ```js
 pug.render('p Hello world!');
 ```
 :::
 
-## Options
+## 选项 ~~ Options
 
-All API methods take the following set of options:
+所有的 API 方法都可以使用以下的选项：
 
 ```parameter-list
 filename
 ~ string
-~ The name of the file being compiled. Used in exceptions, and required for relative includes and extends. Defaults to `'Pug'`.
+~ 要编译的代码的文件名。用于异常信息以及（使用相对路径的）包含（include）和扩展（extend）操作。默认值是 `'Pug'`。
```

Comment:
Use code style for `include` and `extend`. Also note that the proper keyword name is `extends`, not `extend` -- the latter is just an alias but the former is used in this documentation.